gaia-fast-list.js

```diff
@@ -283,10 +312,15 @@ Internal.prototype = {
   createList: function() {
     debug('create list');
     this.fastList = new FastList(this);
-    this.el.classList.add('layerize');
+    setTimeout(() => this.layerize(), 360);
```

Comment:
not sure what will end up being the best performing solution here, but let's land this and follow up on it :)